### PR TITLE
A0-1361: set session keys check

### PR DESCRIPTION
**Pull request by krzysztofziobro**
Adds a session key check to `run_node.sh`:
- When run as a validator it takes mandatory `--stash_account` argument
- Checks if session keys are set for this stash account - uses `cliain`
- If there are, then makes an RPC call to the node, checking if node has access to the keys.
- If not, it will stop the node and print an error.

### CODE REVIEW

**Review by Marcin-Radecki — CHANGES_REQUESTED** [review]
Solid job, just minor comments

File: run_node.sh [hunk 1]
```diff
@@ -130,3 +139,59 @@ then
...
+
+# Pull cliain from ecr
+docker pull "$CLIAIN_IMAGE"
+
+# Try to retrieve set session keys from chain's storage[, remove cliain image]
+SESSION_KEYS_JSON=$(docker run --network="host" "$CLIAIN_IMAGE" next-session-keys \
+    --account-id "$STASH_ACCOUNT" 2> /dev/null)
+
+# Check if there are any session keys set for the specified stash account
+if [[ -n "$SESSION_KEYS_JSON" ]]
```
[the remark this answers is not in the document]
Comment (krzysztofziobro) [reply run_node.sh:160]: The case in which there are no keys set for the account will be handled in the next PR. But I can add a small echo for now, I guess.

**Review by Marcin-Radecki — APPROVED** [review]
Nice!

File: run_node.sh [hunk 2]
```diff
@@ -65,16 +66,24 @@ while [[ $# -gt 0 ]]; do
...
         *)
             echo "Unrecognized command"
             Help
             exit;;
   esac
 done
 
+if [[ -z "$ARCHIVIST" ]]
+then
+    STASH_ACCOUNT=${STASH_ACCOUNT:?'Please specify your stash account_id.'}
```
Comment (piotrMocz) [inline run_node.sh:84]: Maybe we should make this argument mandatory and reflect this in the `usage` docstring?

File: run_node.sh [hunk 3]
```diff
@@ -130,3 +139,72 @@ then
                -d ${ALEPH_IMAGE}
 fi
 
+echo 'Performing session key checks...'
+
+if [[ -n "$ARCHIVIST" ]]
+then
+    echo 'Node run as archivist: no need to check session keys.'
```
Comment (piotrMocz) [inline run_node.sh:146]: We might (and this is mostly thinking out loud) want to replace the 'archivist' with 'rpc node', the name archivist is a little misleading

**Review by piotrMocz — COMMENTED** [review]
Cool! I'd consider making the `stash_account` argument mandatory -- right now the `usage` docstring suggests it's optional.

As a sidenote, I generally like to reference variables as `${VAR}` -- in some cases it can save you some surprises.

**Review by piotrMocz — APPROVED** [review]
Great! One more ask though: could also add the new argument to the README?

**Pull request merged**